mne_connectivity/tests/test_connectivity.py

```diff
@@ -144,9 +144,7 @@ def test_connectivity_containers(conn_cls):
     conn = conn_cls(data=correct_numpy_input, n_nodes=3, **extra_kwargs)
 
     # test that get_data works as intended
-    with pytest.raises(
-        ValueError, match="Invalid value for the " "'output' parameter*."
-    ):
+    with pytest.raises(ValueError, match="Invalid value for the 'output' parameter*."):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match="Invalid value for the 'output' parameter"):
```
probably meant to be `.*` but explicit `.*` is not required at the end of the match string

Reply:
Were a few other cases of this in the same file, so will change those too.